addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -96,7 +96,7 @@
             'info': $o-we-color-info,
             'warning': $o-we-color-warning,
             'danger': $o-we-color-danger,
-            'brand_primary': $o-brand-primary,
+            'brand_primary': $o-we-toolbar-color-accent,
```

Comment:
Hm, not sure about this. @Brieuc-brd wdyt?

Reply:
Could you check with @pierre-pde that this is indeed the desired result?

Reply:
@morm-odoo , thanks for having a look
![image](https://github.com/odoo/odoo/assets/75257428/e9911a52-b889-415f-84b1-b98db2b505e7)

cc @Brieuc-brd  & @edi-odoo